TrafficCapture/captureProtobufs/src/main/proto/TrafficCaptureStream.proto

```diff
@@ -69,6 +69,8 @@ message TrafficObservation {
 message TrafficStream {
   string connectionId = 1; // persistent identifier for all stream objects within a single connection
   string nodeId = 5; // unique namespace such as a nodeId or computer mac address
+  optional int32 requestCount = 6;
```

Comment:
Do we plan on these always being optional or just into we get things in place for TrafficCapture?

Reply:
I think they'll always be optional. In subsequent commits they're optional. With protobuf, an optional value that isn't present in the binary will be interpreted as the default value for the type, meaning 0 and false. That's what I want anyway and lets us remain backward compatible with existing streams.